Makefile

```diff
@@ -8,6 +8,8 @@ IMAGE_TAG = $(VERSION)
 EXTRA_BOOT_PARAMS =
 VARIANT = Kinoite
 WEB_UI = false
+ENROLLMENT_PASSWORD = isogenerator
```

Comment:
I know I suggested "isogenerator" as a more generic default password... but this IS a ublue org action... 

Should it default to "ublue-os" for the sanity of defaulting to what we've put in all our docs?

(More a question for the team)

Reply:
It would save effort in our actions as I wouldn't have to specify it every time.

Reply:
@KyleGospo do you agree `ublue-os` is a better default?

If so, lets change that back as the default.  Please forgive me for causing confusion. And I'll approve then. :-)

Reply:
The rest of the params are defaulting to ublue arguments, so I vote to keep ublue-os.

Reply:
@noelmiller go for it!